src/popover/types.js

```diff
@@ -109,6 +109,8 @@ export type BasePopoverPropsT = {
   showArrow?: boolean,
   /** Whether to toggle the popover when trigger is clicked or hovered. */
   triggerType: TriggerTypeT,
+  /** Margin of the popover */
+  popoverMargin: number,
```

Comment:
Sorry I missed this, we need to make the prop optional because it will fail flow check for existing applications